iOS Client/PodcastsController/PodcastFilterTableViewCell.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  PodcastFilterTableViewCell.swift
+//  iOS Client
+//
+//  Created by Jonathan Witten on 4/9/18.
+//  Copyright © 2018 Mockingbird. All rights reserved.
+//
+
+import UIKit
+
+class PodcastFilterTableViewCell: UITableViewCell {
+
+    @IBOutlet weak var toggleSwitch: UISwitch!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var podcastImage: UIImageView!
+    
+    var podcast: PodcastStream?
+    
+    static var reuseIdentifier: String = "PodcastFilterTableViewCell"
+    func configure(image: UIImage? = nil, podcast: PodcastStream, on: Bool = false) {
```

Comment:
Something I read online reminded me that this pattern is a violation of MVC. By passing the PodcastStream model to the TableViewCell, the view is talking directly to the model.

It does provide a nice way to get this code out of the VC. Idk. In articles, I just set everything on the view from the VC for now. Curious to hear your thoughts.

Reply:
Yeah I see what you're saying. It's just a matter of whether to list out properties one by one in the constructor or just have one for the entire model. I'm ok with changing it

Reply:
Up to you. The VC code could also just set properties directly